src/python/grpcio/_unixccompiler_patch.py

```diff
@@ -32,90 +32,43 @@
 from distutils import ccompiler
 from distutils import errors
 from distutils import unixccompiler
+from distutils import util
 import os
 import os.path
 import shutil
 import sys
 import tempfile
 
+def _unix_commandfile_spawn(unused_self, command, *args, **kwargs):
```

Comment:
It looks like the function signature for ```unixcccompiler.spawn()``` is slightly different: 
https://bitbucket.org/carljm/python-distutils/src/48c42eeaee4410d76675b637bcd401b8919ff19a/ccompiler.py?fileviewer=file-view-default#ccompiler.py-1034

What if instead of delegating to ```util.spawn()```, we delegated to ```super(UnixCCompiler, self).spawn()```
(If you have a more authoritative source than this guys bitbucket that says otherwise, please disregard)